sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -24,6 +26,64 @@ def create_endpoints(plugin):
     plugin.on_message('g#', github_repo_link)
     plugin.on_message('^inspect', inspect, flags=re.IGNORECASE, mention=True, admin=True)
     plugin.on_message('^help', help_message, flags=re.IGNORECASE, mention=True)
+    plugin.on_message(STOCK_REGEX, stock_quote)
+
+
+async def stock_quote(message, app):
+    match = STOCK_REGEX.search(message['text'])
```

Comment:
`message['text']` might raise a `KeyError` or be `None`. I'm not sure exactly when but probably when messages are shared or something like that.

See https://github.com/pyslackers/sirbot-pyslackers/pull/126/files#diff-1025f9d807ea37ea4a63f9b60220d569L186, but there are probably some other handler where I don't check that